packages/components/src/navigator/utils/router.ts

```diff
@@ -21,3 +21,28 @@ export function patternMatch( path: string, screens: Screen[] ) {
 
 	return undefined;
 }
+
+export function findParent( path: string, screens: Screen[] ) {
+	if ( path[ 0 ] !== '/' ) {
+		return undefined;
+	}
+	const pathParts = path.split( '/' );
+	let parentPath;
+	while ( pathParts.length > 1 && ! parentPath ) {
```

Comment:
The fact that the `! parentPath` _implicitly_ checks for an empty string (on top of `undefined`) could bite us back in a a future refactor. Should we make the check more explicit, ie. 

```suggestion
	while ( pathParts.length > 1 && parentPath !== undefined && parentPath !== '' ) {
```